### PR TITLE
Implement `hit load profile`

### PR DESCRIPTION
**Pull request by certik**
And add hit commandline support

@cekees, @dagss this is the most useful part of #325, which I need for my work. It doesn't conflict with any commands and the implementation, while not perfect, is reasonably simple and robust. I want this in, so that I can use it.

The way I use it is that it prints the command to execute in bash to load the bash utilities, and I put it into my .bashrc. Then `hit load profile` works out of the box, all the time.

### CODE REVIEW

File: share/hashdist/setup-env.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# This file is part of Hashdist to setup Hashdist environment for bash.
+#
+# We provide a 'hit' Bash function, that checks for subcommand is called. If it
```
Comment (cekees) [inline share/hashdist/setup-env.sh]: Do you mean 'checks for the subcommand called'?

**Comment by cekees** [comment]
Great! So if I wanted to add csh support I would just add a properly defined `setup-env.csh`  script and  `--print-csh-commands` option, right?

File: share/hashdist/setup-env.sh (hunk 1 above)
Comment (certik) [inline share/hashdist/setup-env.sh]: Yes, indeed. I just fixed it.

**Comment by certik** [comment]
@cekees that's right. And also add this into the help screen, so that the user knows how to load it. One could then factor out some common code, but that is minor. It is setup so that you can add more shells in the future.

**Comment by dagss** [comment]
LGTM.

**Comment by certik** [comment]
Thanks!

**Pull request merged**